core/src/test/java/de/danoeh/antennapod/core/feed/VolumeAdaptionSettingTest.java

```diff
@@ -5,7 +5,7 @@
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
What is the reason for this change?

Reply:
The old assertThat is deprecated, and this is what it wants us to use now.